base/w3ctime.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "std/ctime.hpp"
+#include "std/string.hpp"
+
+namespace base
+{
+
+inline bool NotATime(time_t const t) noexcept { return t == -1; }
+
+// See http://www.w3.org/TR/NOTE-datetime to learn more
+// about a format.
+
+// Parse a string of a format YYYY-MM-DDThh:mmTZD
```

Comment:
Add 's' to all verbs, like "Parses", "Returns", in other words, use Present Simple for all comments here.